COMMUNICATION.md

```diff
@@ -54,7 +54,7 @@ These are paraphrased or downright copied over from Github’s pull request guid
 - Offer ways to simplify or improve code.
 - Avoid using derogatory terms, like “stupid”, when referring to the work someone has produced. Don’t shame, no one started out knowing.
 - Be humble. (“I’m not sure, let’s try…”)
-- NEVER use hyperboles. Unless needed. (“Never store sensitive data in plain text!”)
+- Never deal in absolutes. Unless needed. (“Never store sensitive data in plain text!”)
```

Comment:
This is good! But remove the explanation mark, that way it comes across as an attack.